src/subdomains/core/history/mappers/transaction-dto.mapper.ts

```diff
@@ -266,7 +267,11 @@ function getTransactionStateDetails(entity: BuyFiat | BuyCrypto | RefReward): {
 
       case CheckStatus.FAIL:
         if (entity.chargebackDate) return { state: TransactionState.RETURNED, reason };
-        return { state: TransactionState.FAILED, reason };
+        return {
+          state: TransactionState.FAILED,
+          reason,
+          chargebackTxId: entity.chargebackCryptoTxId ?? entity.chargebackBankTx.txId,
```

Comment:
I would use `entity.chargebackRemittanceInfo` instead of `entity.chargebackBankTx.txId`, or?